drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -633,6 +633,23 @@ static void set_up_fixed_clock_sources(void)
 
 		z_stm32_hsem_unlock(CFG_HW_RCC_SEMID);
 	}
+
+#if defined(RCC_HSI48_SUPPORT)
```

Comment:
Are we sure all series with HSI48 are defining this switch ?

Reply:
better use` #if defined(RCC_CR2_HSI48ON) || defined(RCC_CR_HSI48ON) || defined(RCC_CRRCR_HSI48ON)`